spec/controllers/review_controller_spec.rb

```diff
@@ -10,31 +10,30 @@
 
     describe "submit for 2i" do
       describe "GET submit for 2i page" do
+        required_permissions = ["signin", "GDS Editor", "Unreleased feature"]
+
         it "can be accessed by users with GDS editor and Unreleased feature permissions" do
-          stub_user.permissions = ["signin", "GDS Editor", "Unreleased feature"]
+          stub_user.permissions = required_permissions
           get :submit_for_2i, params: { step_by_step_page_id: step_by_step_page.id }
 
           expect(response.status).to eq(200)
         end
 
-        it "cannot be accessed by users with only GDS editor permissions" do
-          stub_user.permissions << "GDS Editor"
-          get :submit_for_2i, params: { step_by_step_page_id: step_by_step_page.id }
-
-          expect(response.status).to eq(403)
-        end
-
-        it "cannot be accessed by users with neither GDS editor and Unreleased feature permissions" do
-          stub_user.permissions = %w(signin)
-          get :submit_for_2i, params: { step_by_step_page_id: step_by_step_page.id }
+        required_permissions.drop(1).each do |required_permission|
+          it "cannot be accessed by users without the #{required_permission} permissions" do
```

Comment:
Nitpick - I'd make this singular, as it only deals with one permission at a time:

```suggestion
          it "cannot be accessed by users without the #{required_permission} permission" do
```

Same for the ones below.